bin/sample_build.py

```diff
@@ -24,4 +24,6 @@
         check=True,
     )
 
-    sys.exit(subprocess.run([sys.executable, "-m", "cibuildwheel"], cwd=project_dir).returncode)
+    raise SystemExit(
+        subprocess.run([sys.executable, "-m", "cibuildwheel"], cwd=project_dir).returncode
+    )
```

Comment:
This is poor form, IMO. I watched the video and follow the logic, but sys.exit is more idiomatic to me. The `raise` implies an error. I realise that the exception is how Python works internally, but I believe that it does that to conveniently reuse the try/finally semantics, and not because it should be viewed as an error. It comes across strangely in the case that this is `raise SystemExit(0)` - there, the program is exiting successfully but our code is raising an exception. 

More familiar is a function that never returns, like C's `exit()`. That's also how most (all?) other mainstream languages do it, e.g. [Rust](https://doc.rust-lang.org/std/process/fn.exit.html), [Ruby](https://stackoverflow.com/questions/30337972/how-to-tell-a-ruby-program-to-close-itself), [Node.js](https://nodejs.org/api/process.html#processexitcode), [Java](https://www.geeksforgeeks.org/system-exit-in-java/), [Go](https://gobyexample.com/exit)...